src/unified_graphics/etl/diag.py

```diff
@@ -215,7 +220,7 @@ def load(path: Path) -> xr.Dataset:
     return transformed
 
 
-def save(path: Union[Path, str], *args: xr.Dataset):
+def save(session: Session, path: Union[Path, str], *args: xr.Dataset):
```

Comment:
Since this signature is being updated - does the usage in `src/unified_graphics/etl/aws.py` `lambda_handler()` function need to be updated to match?

Reply:
Yeah, it should. I’m troubled that `mypy` didn’t flag this. I see an error in my editor based on the function signature, but when I run `mypy` over the source, it doesn’t flag any errors. 🤔